File: javaScript/BreakfastTask/CalculatePowerOfNumber.java

public class CalculatePowerOfNumber{
public static void main(String... args){


int numberBase = 2;
int number = 5;

int result = 1;

for(int index = 0; index < Math.abs(number); index++){
result *= numberBase;

}

if(number < 0){
result = 1 / result;

}

System.out.println(numberBase + " raised to " + number + " is " + result);


}
}

File: javaScript/Task/calculatePowerOfNumber.js
let numberBase = 2;
let number = 5;
let result = 1;

for (let index = 0; index < Math.abs(number); index++) {
    result *= numberBase;
}
if (number < 0) {
    result = 1 / result;
}
console.log(numberBase + " raised to " + number + " is " + result);